backend/src/main/repositories/EmployeeAccountRepository.ts

```diff
@@ -93,6 +93,28 @@ export default class EmployeeAccountRepository implements CRUD {
     }
   };
 
+  public getEmployeesByRegex = async (email: string): Promise<EmployeeAccount[]> => {
+    try {
+      const operatorsAliases = {
+        like: Op.like,
+      };
+      const data = await EmployeeAccount.findAll({
+        limit: 5,
+        where: {
+          email: {
+            [operatorsAliases.like]: `${email}%`,
+          },
+        },
+      });
+      return Promise.resolve(data);
+    } catch (err: any) {
+      if (err instanceof BaseError) {
+        throw new Error(`${err.name}, message: ${err.message}`);
```

Comment:
same comment as before, I think you should reject instead of throw.

Reply:
Great